src/components/Tabs/Tabs.tsx

```diff
@@ -296,8 +296,8 @@ export const Tabs = ({
                                     return (
                                         <button
                                             className={merge([
-                                                'tw-flex tw-items-center tw-w-full tw-mb-3 tw-text-left tw-text-text-weak',
-                                                tab.disabled && 'tw-text-text-disabled',
+                                                'tw-flex tw-items-center tw-w-full tw-mb-3 tw-text-left',
```

Comment:
@mnoleto You are making the change on the overflow menu here. I think the issue is on line `232` to`234`
```
tab.disabled && 'tw-text-text-disabled',
!tab.disabled && 'hover:tw-text-text',
tab.id === activeItemId ? 'tw-font-medium tw-text-text' : 'tw-text-text-weak',